build.gradle.kts

```diff
@@ -140,3 +140,55 @@ val disableSigning by tasks.registering {
 }
 
 tasks.getByName("publishToMavenLocal").dependsOn(disableSigning)
+
+/**
+ * Downloads the BCP 47 Language Tag Registry as defined by iana,
+ * @see <a href="https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry">https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry</a>,
+ * is JSON format.
+ * The types which are required for the validation of the Locale Constraint, are extracted and written to a javascript file.
+ */
+val downloadBCP47LanguageTagRegistry = tasks.register( "downloadBCP47LanguageTagRegistry" ) {
```

Comment:
Please turn this task implementation into a separate source file (which can be Java instead of Kotlin) in the [buildSrc directory structure](https://docs.gradle.org/current/userguide/organizing_gradle_projects.html#sec:build_sources). This will make it easier and cleaner to add further custom build task implementations (e.g. for generating other artifacts) in the future.